File: nodejs/src/handlers/authenticate_platform.ts

/** 
  * Use OAuth2 to authenticate into a platform.
  * 
  * Currently just logs the event.
  * 
  * @param: event - TODO
  * @param: context - Lambda execution context. Not needed.
*/
exports.authenticate_platform_handler = async function(event, context) {
  console.log("Authenticate Platform Event: \n" + JSON.stringify(event, null, 2))
  return {statusCode: 200, body: context.logStreamName}
}

File: python/src/handlers/authenticate_platform.py
import json
import logging
from typing import Any, Dict

logger = logging.getLogger("authenticate_platform_handler")


def authenticate_platform_handler(event: Dict, context: Any) -> str:
    """Authenticate a user with a given platform.

    Currently empty - will implement in different PR.
    """
    logger.info(f"Authenticate platform event: {json.dumps(event)}")
    return "foo"
